tox.ini

```diff
@@ -22,6 +22,7 @@ conda_deps=
     s3fs
     pyftpdlib
     cloudpickle
+    smbprotocol
```

Comment:
I think this just needs to be moved from the `conda_deps` list here to the `deps` list below

Reply:
I'll fix and push a new commit.